Bottom: acd48d2860fb156df3108eb275dce5d349142dde
Top:    a0212bfa6d5798b9061c238fea029da8d78ad0da
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-11-15 01:53:38 +0000

Refresh of add-raw-polygons-support-for-t

---

diff --git a/src/hid/common/hidgl.c b/src/hid/common/hidgl.c
index 73bf7d0..413d92f 100644
--- a/src/hid/common/hidgl.c
+++ b/src/hid/common/hidgl.c
@@ -38,6 +38,7 @@
 #include "hidgl.h"
 #include "rtree.h"
 
+
 #ifdef HAVE_LIBDMALLOC
 #include <dmalloc.h>
 #endif
